docs/flink-ddl.md

```diff
@@ -159,29 +162,16 @@ Table create commands support the commonly used [Flink create clauses](https://n
 * `COMMENT 'table document'` to set a table description.
 * `WITH ('key'='value', ...)` to set [table configuration](../configuration) which will be stored in Iceberg table properties.
 
-Currently, it does not support computed column, primary key and watermark definition etc.
-
-### `PARTITIONED BY`
```

Comment:
I don’t recommend deleting this part, we can configure it as a child of the cteate table like spark DDL does.